configuration/restdocs.md

```diff
@@ -67,6 +67,27 @@ The commands above have been copied from the REST API documentation for illustra
 You can try and validate rest api calles from within the openHAB UI.
 Just log in with an admin user, navigate to `Developer Tools -> API Explorer` and start exploring.
 
+## Authentication
+
+Starting with version 3, openHAB supports password protection for sensible contents such as parts of the semantic model.
```

Comment:
Why was the "supports password protection for sensible contents **such as parts of the semantic** model" added? I can not find anything about changing the protection of **parts** of the semantic model.

As far as I understand, the authentication works in the following way:
1. Sensitive things that can only be changed by an "Admin" user need authentiction of any "Admin" user.
2. Not sensitive things are by default accessible without any authentication.
3. If the "API Security -> Implicit User Role" is disabled, then authentication is needed by any "Admin" or "Users" user.

I.e., Users will always have access to all the Items. This statement also seems consistent with various explanations on the forums.

So why was this "**parts** of the semantic" model added here? As the semantic model contains items, and items can always all be accessed, why was this written? Should it be changed?

Reply:
Probably it should be revised; I'm out of openHAB for two years and can't even remember writing this PR 😅 Feel free to submit a more precise explanation ...

Reply:
No worries :)

I will probably post something in the forum and get this revised.